src/librustc_mir/dataflow/mod.rs

```diff
@@ -650,6 +650,20 @@ pub trait BottomValue {
     const BOTTOM_VALUE: bool;
 
     /// Merges `in_set` into `inout_set`, returning `true` if `inout_set` changed.
+    ///
+    /// You usually don't need to override this, since it automatically applies
```

Comment:
I think you can be stronger here: "It is almost certainly wrong to override this".